.github/ISSUE_TEMPLATE/NEW_EXPERIMENT.md

```diff
@@ -0,0 +1,85 @@
+---
+name: "\U0001F4CB New experiment proposal"
+about: "Proposal and workflow guideline for adding a new experiment \U0001F52C."
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- ⚠️ If you do not respect this template, your issue will be closed -->
+<!-- ⚠️ Make sure to browse the opened and closed issues -->
+
+# New experiment proposal
+Provide the experiment and implementation details below and ask the qiskit-experiments core team to review 
+your proposal. 
+
+## General details
+
+### Experiment name
+What is the experiment class name? This name will also be used in the API documentation and tutorial.
+
+### Experiment type
+What is the experiment type? Characterization, calibration, verification, validation, or other? 
+
+### Experiment protocol
+Provide a concise description of the experiment. Make sure you cover the following aspects:  
+* What is the main goal of the experiment?
+* Describe the circuits/pulses and their main parameters needed for the experiment
+* What are the main outputs of the experiment?
+
+### Experiment analysis
+Provide a concise description of the experiment's analysis. Make sure you cover the following aspects:
+* What is the main fit model for the experiment? 
+* What are the main fit parameters? Specify parameter defaults and bounds where relevant
+* How would you evaluate whether the fit is good or bad?
+
+### References 
+If applicable, provide a few of the most relevant references here.
+
+## Implementation details
+Provide additional details pertaining to the implementation of the experiment and its analysis. Use the following as 
+guiding points. Provide code snippets and usage examples where appropriate. 
+### Experiment implementation
+* What are the base classes for the experiment and its analysis?
+* List the input parameters, required and optional, and types of each
+* Experiment options and default values (e.g. default transpile and run options)
+* Are there any limitations or special requirements for running the experiment as part of a composite experiment? 
+Provide details.
+
+### Experiment analysis
+* Analysis options and default values
+* How does it generate initial guesses
+* What plots will be generated? 
+
+---
+
+# Workflow
+These are the steps required for adding a new experiment to qiskit-experiments. It is advisable to follow this workflow
+in order and get the proper review approvals from the qiskit-experiments core team before moving on to subsequent steps 
+in the workflow. 
+
+NOTE: An experiment PR failing to complete all the workflow steps below will not be merged!
+
+- [ ] Complete the proposal section above and have your proposal reviewed and approved 
```

Comment:
Since the proposal is an issue, what is the approval process? How does the author know it is safe to begin implementing the experiment? Maybe adding a `experiment-approved` tag to the issue (currently, I think anyone can add tags to issues but I see that there is a beta feature for protected tags that only maintainers can add).

Reply:
I assumed that once the author completes the proposal description, he or she will request a review (as stated in the first step here). Then there will probably be some interchange with the package reviewers at the end of which they will say something like 'looks good, let's have this experiment added'. Adding an `approved` label would be nice addition, agreed. Do you think we should explicitly write that the authors should make sure they get such labeling?

Reply:
It is one idea. There might be another way. I think it would be nice to have something that is clear since in an issue discussion there could be some ambiguity.